app/services/backup_service.rb

```diff
@@ -41,8 +41,11 @@ def build_outbox_json!(file)
           end
         end
 
-        Oj.dump(item)
-      end.join(','))
+        json_str = Oj.dump(item, indent: 2)
+        json_str.lines.map do |line|
+          "    #{line.chomp}"
+        end.join("\n")
```

Comment:
Are there any config-level or options-level ways to do this so we could tell the json dumping lib what to do, instead of the manual formatting?

Separately - total side note to this PR, but there's an ongoing discussion about switching from `oj` to `json` gem - this would need rebase after that. Though that conversation is still in progress, so not sure on timing.

Reply:
The cleaner way to do all of this is to assemble the entire JSON structure into a Hash, then dump all at once. 😌

However, that would require both:

- more code changes than I'm comfortable with in a single PR
- a likely performance hit

Tbh I'm not very happy with this code for other non-functional reasons. Unless you object, I'll close the PR! 😅

Reply:
No objection - I think if something like this could be very cleanly done through basically config-only changes, it'd be harmless and maybe useful for some people.